src/applications/mhv/medical-records/containers/AllergyDetails.jsx

```diff
@@ -33,6 +33,7 @@ import { txtLine } from '../../shared/util/constants';
 const AllergyDetails = props => {
   const { runningUnitTest } = props;
   const allergy = useSelector(state => state.mr.allergies.allergyDetails);
+  const allergyList = useSelector(state => state.mr.vaccines.allergyList);
```

Comment:
This is not pulling the right state. It should be `state.mr.allergies.allergiesList`

Reply:
This issue is fixed.